src/STM32RTC.h

```diff
@@ -175,26 +184,32 @@ class STM32RTC {
     void setDate(uint8_t day, uint8_t month, uint8_t year);
     void setDate(uint8_t weekDay, uint8_t day, uint8_t month, uint8_t year);
 
-    void setAlarmSubSeconds(uint32_t subSeconds);
-    void setAlarmSeconds(uint8_t seconds);
-    void setAlarmMinutes(uint8_t minutes);
-    void setAlarmHours(uint8_t hours, AM_PM period = AM);
-    void setAlarmTime(uint8_t hours, uint8_t minutes, uint8_t seconds, uint32_t subSeconds = 0, AM_PM period = AM);
+    void setAlarmSubSeconds(uint32_t subSeconds, Alarm name = ALARM_A);
+    void setAlarmSeconds(uint8_t seconds, Alarm name = ALARM_A);
+    void setAlarmMinutes(uint8_t minutes, Alarm name = ALARM_A);
+    void setAlarmHours(uint8_t hours, Alarm name);
+    void setAlarmHours(uint8_t hours, AM_PM period = AM, Alarm name = ALARM_A);
+    void setAlarmTime(uint8_t hours, uint8_t minutes, uint8_t seconds, Alarm name);
```

Comment:
else it is ambiguous with the full one.
If name was defined by default, doing this will produced an ambiguous called:
`rtc.setAlarmTime(hours, minutes, seconds);`
The goal is to provide API without optional arguments.